ang/test/civicase/DateHelperService.spec.js

```diff
@@ -18,16 +19,26 @@
 
       describe('isOverdue()', function () {
         beforeEach(function () {
-          pastDate = moment().subtract(7, 'd');
-          futureDate = moment().add(7, 'd');
+          activitiyWithBooleanOverdue = activitiesMockData[0];
```

Comment:
You are not checking the false values. Newly created variables should not be global. As it is used only inside `isOverdue` test.